structure_tests/ext_run.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+
+command -v docker > /dev/null 2>&1 || { echo "Docker is required to run GCP structure tests, but is not installed on this host."; exit 1; }
+
+command docker ps > /dev/null 2>&1 || { echo "Cannot connect to the Docker daemon!"; exit 1; }
+
+usage() {
+	echo "Usage: $0 [-i <image>] [-c <config>] [-v] [-e <entrypoint>]"
+	exit 1
+}
+
+export VERBOSE=0
+export CMD_STRING=""
+export ENTRYPOINT="./test/structure_test"
+export ST_IMAGE="gcr.io/nick-cloudbuild/structure_test"
+
+declare -i CONFIG_COUNTER=0
+
+mkdir /tmp/st_configs
+
+while test $# -gt 0; do
+	case "$1" in
+		--image|-i)
+			shift
+			if test $# -gt 0; then
+				export IMAGE_NAME=$1
+			else
+				usage
+			fi
+			shift
+			;;
+		--verbose|-v)
+			export VERBOSE=1
+			shift
+			;;
+		--entrypoint|-e)
+			shift
+			if test $# -gt 0; then
+				export ENTRYPOINT=$1
+			else
+				usage
+			fi
+			shift
+			;;
+		--config|-c)
+			shift
+			if test $# -eq 0; then
+				usage
+			else
+				cp "$1" /tmp/st_configs/cfg_$CONFIG_COUNTER.json
```

Comment:
Some comments here would be nice. I understand why we're doing the copy, but it might be hard to understand later.